src/NUnitFramework/tests/Constraints/DictionaryContainsKeyConstraintTests.cs

```diff
@@ -448,6 +449,7 @@ public class TestNonGenericDictionary : IDictionary
             public TestNonGenericDictionary(int key)
             {
                 _key = key;
+                Keys = new[] { key };
```

Comment:
Note to reviwer: Our custom `IDictionary` implementation for testing was previously throwing a `NullReferenceException` if you tried to reference the `Keys` property of it.